libs/designsystem/src/lib/components/segmented-control/segmented-control.component.spec.ts

```diff
@@ -165,3 +167,80 @@ describe('SegmentedControlComponent', () => {
     });
   });
 });
+
+describe('SegmentedControl with Badge', () => {
+  let itemsWithBadge: SegmentItem[] = [
+    {
+      text: 'First Item',
+      id: 'first',
+      badge: {
+        content: '1',
+        themeColor: 'danger',
+      },
+    },
+    {
+      text: 'Second item',
+      id: 'second',
+      badge: {
+        icon: 'attach',
+        themeColor: 'danger',
+      },
+    },
+    {
+      text: 'Third item',
+      id: 'third',
+      badge: {
+        content: '1',
+        icon: 'attach',
+        themeColor: 'danger',
+      },
+    },
+  ];
+
+  let spectator: SpectatorHost<SegmentedControlComponent>;
+
+  const createHost = createHostFactory({
+    component: SegmentedControlComponent,
+    declarations: [MockComponents(ChipComponent, BadgeComponent, IconComponent)],
+    imports: [TestHelper.ionicModuleForTest],
+  });
+
+  beforeEach(() => {
+    spectator = createHost(
+      `<kirby-segmented-control>
+       </kirby-segmented-control>`
+    );
+  });
+
+  it('should render a badge per item that has it defined', () => {
```

Comment:
Wording seems off 🤨? 
Perhaps "Should render a badge per item it has defined"?

Reply:
Makey da sense, corrected it!